implementations/rust/ockam/ockam_api/src/kafka/key_exchange/controller.rs

```diff
@@ -170,3 +283,209 @@ impl KafkaKeyExchangeControllerImpl {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::kafka::key_exchange::controller::KafkaKeyExchangeControllerImpl;
+    use crate::kafka::{ConsumerPublishing, ConsumerResolution};
+    use crate::test_utils::{AuthorityConfiguration, TestNode};
+    use ockam::identity::Identifier;
+    use ockam_abac::{Action, Env, Resource, ResourceType};
+    use ockam_core::compat::clock::test::TestClock;
+    use ockam_multiaddr::MultiAddr;
+    use std::sync::Arc;
+    use std::time::Duration;
+    use tokio::runtime::Runtime;
+    use tokio::time::timeout;
+
+    #[test]
+    pub fn rekey_rotation() -> ockam_core::Result<()> {
+        let runtime = Arc::new(Runtime::new().unwrap());
+        let runtime_cloned = runtime.clone();
+        std::env::set_var("OCKAM_LOG", "none");
```

Comment:
"OCKAM_LOG" is not used anymore, you can write `std::env::set_var("OCKAM_LOGGING", "false");` instead.
I'm going to do a PR to fully remove it from the code and comments.